content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -14,62 +14,68 @@ weight: 270
 
 ## {{% heading "prerequisites" %}}
 
-You need to have a Kubernetes cluster, and the kubectl command-line tool must
-be configured to communicate with your cluster. It is recommended to follow this
-guide on a cluster with at least two nodes that are not acting as control plane
-nodes. If you do not already have a cluster, you can create one by using
-[minikube](https://minikube.sigs.k8s.io/docs/tutorials/multi_node/).
+Before you follow steps in this page to deploy, manage, back up or restore etcd,
+you need to understand the typical expectations for operating an etcd cluster.
+Refer to the [etcd documentation](https://etcd.io/docs/) for more context.
 
-### Understanding etcdctl and etcdutl
+Key details include:
 
-`etcdctl` and `etcdutl` are command-line tools used to interact with etcd clusters, but they serve different purposes:
-
-- `etcdctl`: This is the primary command-line client for interacting with etcd over a
-network. It is used for day-to-day operations such as managing keys and values,
-administering the cluster, checking health, and more.
-
-- `etcdutl`: This is an administration utility designed to operate directly on etcd data
-files, including migrating data between etcd versions, defragmenting the database,
-restoring snapshots, and validating data consistency. For network operations, `etcdctl`
-should be used.
-
-For more information on `etcdutl`, you can refer to the [etcd recovery documentation](https://etcd.io/docs/v3.5/op-guide/recovery/).
-
-<!-- steps -->
-
-## Prerequisites
-
-* Run etcd as a cluster of odd members.
+* The minimum recommended etcd versions to run in production are `3.4.22+` and `3.5.6+`.
 
 * etcd is a leader-based distributed system. Ensure that the leader
   periodically send heartbeats on time to all followers to keep the cluster
   stable.
 
-* Ensure that no resource starvation occurs.
+* You should run etcd as a cluster with an odd number of members.
+
+* Aim to ensure that no resource starvation occurs.
```

Comment:
"aim to" duplicates "ensure"?

Reply:
Bare “ensure” would mean: you MUST prevent resource starvation; the Kubernetes project insists that you do.
“Aim to ensure” means: invest effort in preventing resource starvation.